src/Option.js

```diff
@@ -64,6 +64,7 @@ class Option extends React.Component {
 
 	onFocus (event) {
 		if (!this.props.isFocused) {
+			this.props.onOptionRef(this.ref);
```

Comment:
Given the rest of the changes, this would be clearer if we just passed the ref as a third argument to the `onFocus` handler (below) and made the corresponding update in Select.js